ffi/src/proofs/range.rs

```diff
@@ -16,7 +16,7 @@ use crate::{
 pub struct CreateRangeProofArgs<'a> {
     /// The root hash of the revision to prove. If `None`, the latest revision
     /// is used.
-    pub root: Maybe<BorrowedBytes<'a>>,
+    pub root: BorrowedBytes<'a>,
```

Comment:
re your try_into; it can be removed if this is a `HashKey` (`use crate::HashKey;`).